docs/guide/intro/index.adoc

```diff
@@ -506,7 +506,16 @@ Each artifact version is the one that would get installed when building the Boot
 
 The Maven artifacts to upgrade must be added to the `<dependencies>` of your Maven project (with a `provided` scope 
 to avoid the dependency to be added to your application). In addition, the plugin configuration element
-`<overridden-server-artifacts>` must contain the artifacts. For example, the version 3.0.0 of `io.undertow:undertow-core` will 
+`<overridden-server-artifacts>` must contain the artifacts.
+
+In order to narrow artifact upgrade inside a Galleon feature-pack, the `<FeaturePack>` configuration element is evolved 
```

Comment:
This may need to read "In order to narrow an artifact upgrade". Inserting the "an" is all.